pkg/interface/src/views/apps/notifications/notification.tsx

```diff
@@ -89,11 +88,22 @@ function NotificationWrapper(props: {
     return api.hark[func](notif);
   }, [notif, api, isMuted]);
 
+  const [visible, setVisible] = useState(false);
```

Comment:
pendanticism, but this is a good candidate for a custom hook. 
.e.g.
```typescript
const { hovering, bind } = useHovering();
// in component, bind is simply { onMouseEnter, onMouseLeave }
<Box {...bind}>

Reply:
Great idea; I wondered how I should do this. I've added another commit which adds and implements this.